submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx

```diff
@@ -1,10 +1,236 @@
-import { createFileRoute } from "@tanstack/react-router";
+import { createFileRoute, Link } from "@tanstack/react-router";
+import {
+  Box,
+  Button,
+  Chip,
+  Container,
+  Divider,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@mui/material";
+import { styled } from "@mui/system";
+import { BCDesignTokens } from "epic.theme";
+import { ProjectStatus } from "@/components/registration/addProjects/ProjectStatus";
+import { PROJECT_STATUS } from "@/components/registration/addProjects/ProjectCard/constants";
+import AddIcon from "@mui/icons-material/Add";
+import SubmissionTable from "@/components/Projects/SubmissionTable";
+import { Plan } from "@/models/Plan";
+
+const HEADER_HEIGHT = 54;
+
+const CardInnerBox = styled(Box)({
+  display: "flex",
+  alignItems: "flex-start",
+  justifyContent: "center",
+  flexDirection: "column",
+  height: "100%",
+  padding: "0 12px",
+});
 
 export const Route = createFileRoute("/_authenticated/_dashboard/projects/")({
   component: ProjectsPage,
   meta: () => [{ title: "Projects" }],
 });
 
 function ProjectsPage() {
-  return <div>Projects Page</div>;
+  const plans: Array<Plan> = [
```

Comment:
please move this into a constants file, also setup a tanstack query api call that just returns this instead of making an actual call